### PR TITLE
#60618099 email verification link

### PR DESCRIPTION
**Pull request by Phunmbi**
### What does this PR do?
This pull request makes it possible for users to get an email that allows them to verify their accounts upon signup.

#### Description of Task to be completed?
- Have the `/api/auth/verification/:verificationToken` endpoint working properly.
- Update Swagger documentation with new routes.
- Set up tests for email verification.

#### How should this be manually tested?
- Setup the repo
- run `npm install`
- run `npm run test-migrate` to automatically migrate your test database
- run `npm run test`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#160618099](https://www.pivotaltracker.com/story/show/160618099)

#### Screenshots (if appropriate)
N/A

#### Questions:
None

### CODE REVIEW

File: controllers/userController.js [hunk 1]
```diff
@@ -41,6 +45,7 @@ module.exports = {
         })
         .catch(() => res.status(500).send({ error: 'Internal server error' }));
     } catch (err) {
+      console.log(err);
```
Comment (AdinoyiSadiq) [inline controllers/userController.js]: Hi. Could you take out this console log? We might also enable the linter to flag future log statements.

File: lib/verificationMail.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  <p>Welcome to Authors Haven!!</p>
+  <br>
+  <br>
+  <br>
+  <p>Verify your account here http://localhost:3000/api/auth/verification/${encryptedToken}</p>
+</div>
+`;
+};
+
+module.exports = verificationMail;
```
Comment (AdinoyiSadiq) [inline lib/verificationMail.js]: Could we use the es6 to export this function?

File: controllers/userController.js (hunk 1 above)
Comment (Phunmbi) [reply controllers/userController.js]: Okay, I'll get on this right now.

File: lib/verificationMail.js (hunk 2 above)
Comment (Phunmbi) [reply lib/verificationMail.js]: Done

**Pull request merged**